app/tests/spec/views/sign_up.js

```diff
@@ -624,6 +626,30 @@ function (chai, _, $, moment, sinon, p, View, Session, AuthErrors, Metrics,
           });
       });
 
+      it('passes the customize sync option to the fxa-client', function () {
+        relier.set('service', 'sync');
+        relier.set('customizeSync', true);
+
+        sinon.stub(view.fxaClient, 'signUp', function () {
+          return p({
+            verified: true
+          });
+        });
+
+        sinon.stub(broker, 'afterSignIn', function () {
+          return p();
+        });
+
+        return view.render()
+          .then(function () {
+            fillOutSignUp(email, 'password', { year: CURRENT_YEAR - 14, context: view });
+            return view.submit();
+          })
+          .then(function () {
+            assert.isTrue(view.fxaClient.signUp.calledWith(email, 'password', relier,
```

Comment:
@zaach, this is good to check, but ultimately, we want to make sure that the `account` object passed to the broker during `beforeSignUpConfirmationPoll` or `afterSignIn` has the `customizeSync` flag set appropriately.